docs/developer_documentation/app-development.md

```diff
@@ -1,106 +1,121 @@
-- [Create a freemarker-template](#create-a-freemarker-template)
-- [Upload your app](#upload-your-app)
-- [Active the app](#active-the-app)
+## Archive structure
+If you downloaded the example archive, you probably noticed that the structure of the archive is pretty straight forward.
 
-## Create a project
-You can create a javascript project in a language you like. 
-We develop our own apps in VueJS so most documentation will point to the VueJS-configuration.
+```
+|-- config.json
+|-- index.html
+|-- js
+    |-- example.js
+|-- css
+    |-- example.css
+```
+
+The index.html is your main entry point, and the contents will be interpreted and rendered by a FreeMarker engine. 
+Relative to your index.html you can include js and css. 
 
-## Update your build configuration
-You must update your build configuration to build a zip-file which you can upload in MOLGENIS. 
+##### Note
+If you use absolute paths, MOLGENIS is unable to serve your resources properly
 
-### webpack
-In your webpack production build configuration
+### Example
+The following example assumes we have a setup according to the tree structure shown in the previous paragraph.
 
+_index.html_
+```html
+<link href="css/example.css"
+<div class="container">
+    <h1>This is an example of including resources with relative paths</h1>
+</div>
+<script src="js/example.js"></script>
+```
+
+_example.js_
 ```js
-  ...
-  assetsRoot: path.resolve(__dirname, '../dist/static'),
-  assetsSubDirectory: '',
-  assetsPublicPath: '/apps/${app.id}',
-  ...
+alert('this is a wonderful example')
 ```
 
-## Update your language specific configuration
-You have to update some configuration in your app to make it work in MOLGENIS. 
+_example.css_
+```css
+.container {
+    background: blue;
+    height: 400px;
+    width: 100%
+}
+```
 
-### VueJS
+## Configuration
+The config.json is a small configuration file that will tell MOLGENIS about your app. 
+It contains the following parameters
+
+| Parameter | type | mandatory | Description |
+| ----------|-------------|-------------|-------------|
+| label     |string| yes| The label used to name your app in the app manager screen |
+| description |string| yes | A description for your app. Useful for making your app more findable in the app manager |
+| version | string |yes |The version of your app. Used to manage app version, updating etc etc.. |
+| apiDependency | string |yes | The version of the MOLGENIS REST api you used in your app. Can be used to give warnings about possible incompatibility with a specific version of MOLGENIS |
+| uri | string |yes | the URL on which the app will be deployed once activated. Best if you make it a simple word without weird characters |
+| includeMenuAndFooter | boolean |yes | If you want to use the MOLGENIS menu, you can this value to __true__ |
+| runtimeOptions | object |no | A map containing initial parameters you might want to give to your app on init |
+
```

Comment:
Note that you can not create a config.json on your root path of the project. It will not build anymore.

Reply:
done